packages/babel-plugin-transform-runtime/scripts/build-dist.js

```diff
@@ -0,0 +1,158 @@
+"use strict";
+
+const path = require("path");
+const outputFile = require("output-file-sync");
+const helpers = require("@babel/helpers");
+const babel = require("@babel/core");
+const template = require("@babel/template");
+const t = require("@babel/types");
+
+const transformRuntime = require("../");
+const corejs2Definitions = require("../lib/definitions").default;
+
+writeHelpers("@babel/runtime");
+
+writeCoreJS2("@babel/runtime-corejs2");
+writeHelpers("@babel/runtime-corejs2", { corejsVersion: 2 });
+
+function writeCoreJS2(runtimeName) {
+  const pkgDirname = getRuntimeRoot(runtimeName);
+
+  const paths = ["is-iterable", "get-iterator"];
+
+  Object.keys(corejs2Definitions.builtins).forEach(key => {
+    const path = corejs2Definitions.builtins[key];
+    paths.push(path);
+  });
+
+  Object.keys(corejs2Definitions.methods).forEach(key => {
+    const props = corejs2Definitions.methods[key];
+    Object.keys(props).forEach(key2 => {
+      paths.push(props[key2]);
+    });
+  });
+
+  paths.forEach(function(corePath) {
+    outputFile(
+      path.join(pkgDirname, "core-js", `${corePath}.js`),
+      `module.exports = require("core-js/library/fn/${corePath}");`
+    );
+  });
+}
+
+function writeHelpers(runtimeName, { corejsVersion } = {}) {
+  writeHelperFiles(runtimeName, { corejsVersion, esm: false });
+  writeHelperFiles(runtimeName, { corejsVersion, esm: true });
+}
+
+function writeHelperFiles(runtimeName, { esm, corejsVersion }) {
+  const pkgDirname = getRuntimeRoot(runtimeName);
+
+  for (const helperName of helpers.list) {
+    const helperFilename = path.join(
+      pkgDirname,
+      "helpers",
+      esm ? "es6" : "",
```

Comment:
if the option is named `esm` then this directory should be IMHO named `esm` too, `es6` makes it more confusing because othen than module format those files wont be written using es6 syntax

Reply:
Sounds good to me. This was just copied over as it was from `babel/runtime`